GameData/RealismOverhaul/RO_SuggestedMods/SystemHeat/HeatControlMicrochannelRadiators.cfg

```diff
@@ -0,0 +1,296 @@
+		SUBTYPE
+		{
+			name = Double
+			title = #LOC_HeatControl_switcher_size_double_title
+			descriptionDetail = #LOC_SystemHeat_switcher_size_double_detail
+			descriptionSummary  = #LOC_HeatControl_switcher_size_double_summary
+			transform = SQUAREPANELDOUBLE
+			addedMass = 0.675
+			MODULE
+			{
+				IDENTIFIER
+				{
+					name = ModuleSystemHeatRadiator
+				}
+				DATA
+				{
+					temperatureCurve
+					{
+						key = 0 0
+						key = 1300 4250
+					}
+				}
+			}
+		}
+		SUBTYPE
+		{
+			name = Square
+			title = #LOC_HeatControl_switcher_size_square_title
+			descriptionDetail = #LOC_SystemHeat_switcher_size_square_detail
+			descriptionSummary  = #LOC_HeatControl_switcher_size_square_summary
+			transform = SQUAREPANELFULL
+			defaultSubtypePriority = 50
+			addedMass = 0
+			MODULE
+			{
+				IDENTIFIER
+				{
+					name = ModuleSystemHeatRadiator
+				}
+				DATA
+				{
+					temperatureCurve
+					{
+						key = 0 0
+						key = 1300 2125
+					}
+				}
+			}
+		}
+		SUBTYPE
+		{
+			name = TwoThirds
+			title = #LOC_HeatControl_switcher_size_twothirds_title
+			descriptionDetail = #LOC_SystemHeat_switcher_size_twothirds_detail
+			descriptionSummary  = #LOC_HeatControl_switcher_size_twothirds_summary
+			transform = SQUAREPANELTWOTHIRD
+			addedMass = -0.225
+			MODULE
+			{
+				IDENTIFIER
+				{
+					name = ModuleSystemHeatRadiator
+				}
+				DATA
+				{
+					temperatureCurve
+					{
+						key = 0 0
+						key = 1300 1420
+					}
+				}
+			}
+		}
+		SUBTYPE
+		{
+			name = Half
+			title = #LOC_HeatControl_switcher_size_half_title
+			descriptionDetail = #LOC_SystemHeat_switcher_size_half_detail
+			descriptionSummary  = #LOC_HeatControl_switcher_size_half_summary
+			transform = SQUAREPANELHALF
+			addedMass = -0.3375
+			MODULE
+			{
+				IDENTIFIER
+				{
+				name = ModuleSystemHeatRadiator
+				}
+				DATA
+				{
+					temperatureCurve
+					{
+						key = 0 0
+						key = 1300 1075
+					}
+				}
+			}
+		}
+		SUBTYPE
+		{
+			name = Third
+			title = #LOC_HeatControl_switcher_size_third_title
+			descriptionDetail = #LOC_SystemHeat_switcher_size_third_detail
+			descriptionSummary  = #LOC_HeatControl_switcher_size_third_summary
+			transform = SQUAREPANELTHIRD
+			addedMass = -0.4455
+			MODULE
+			{
+				IDENTIFIER
+				{
+					name = ModuleSystemHeatRadiator
+				}
+				DATA
+				{
+					temperatureCurve
+					{
+						key = 0 0
+						key = 1300 710
+					}
+				}
+			}
+		}
+	}
+}
+
+// Nonrectangular
+@PART[radiator-microchannel-fixed-2]:AFTER[SystemHeat]
+{
+	%RSSROConfig = True
+	@MODULE[ModuleSystemHeatRadiator]
+	{
+		//This is just a passive radiator. Coolant pumping should be negligable and/or already included in the reactor/converter power
+		//Remove electric power drain
+		!RESOURCE {}
+		
+		//Since this radiator is passive, it can only cool things hotter than it is
+		@overcoolFactor = 1
+
+	!MODULE[ModuleB9PartSwitch] {}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = sizeSwitch
```

Comment:
Are we certain there is exactly 1 ModuleB9PartSwitch on these parts, or should this delete using `:HAS`?

Reply:
Nertea himself does the exact same thing in the SystemHeat patches for HeatControl. There is only one MB9PS present.